cypress/support/step_definitions/dynamicCalls.md

```diff
@@ -1,7 +1,35 @@
+# Table of contents:
+[DynamicCallsGlue](#dynamiccallsglue)
+  - [Dynamic Firebolt Objects](#dynamic-firebolt-objects)
+  - [we test the '(.+)' getters and setters(?: '(.*?)'(?: to '(.*?)')?)?](#we-test-the--getters-and-setters--to-)
+    - [Examples](#examples)
```

Comment:
I think we can just have the top level bullets in the TOC

Reply:
addressed the comments